metaflow/cli.py

```diff
@@ -998,11 +1008,9 @@ def start(
         deco_options,
     )
 
-    # In the case of run/resume, we will want to apply the TL decospecs
-    # *after* the run decospecs so that they don't take precedence. In other
-    # words, for the same decorator, we want `myflow.py run --with foo` to
-    # take precedence over any other `foo` decospec
-    ctx.obj.tl_decospecs = list(decospecs or [])
+    # run/resume can add more decorators with --with, which take precedence
```

Comment:
am fine with this (and associated change) but would re-add the comment about how precedence is determined by order right now (first one wins) so we need to save to apply in before_run. (ie: this comment is correct but doesn't say why we do it this way).